src/modules/ie-objects/elasticsearch/queryBuilder.ts

```diff
@@ -243,7 +259,29 @@ export class QueryBuilder {
 			},
 		];
 
-		if (user.isKeyUser && !isNil(user.sector)) {
+		if (isConsultableRemote) {
+			checkSchemaLicenses = [
+				...checkSchemaLicenses,
+				{
+					terms: {
+						schema_license: [IeObjectLicense.BEZOEKERTOOL_CONTENT],
+					},
+				},
+			];
+		}
+
+		if (isConsultableMedia) {
+			checkSchemaLicenses = [
+				...checkSchemaLicenses,
+				{
+					terms: {
+						schema_license: [IeObjectLicense.BEZOEKERTOOL_CONTENT],
+					},
+				},
+			];
+		}
```

Comment:
Deze filtering lijkt niet te matchen met de spec in de story
https://meemoo.atlassian.net/wiki/spaces/HA2/pages/3705602135/Filters

![image](https://user-images.githubusercontent.com/1710840/222682217-84e4c3b9-25fa-47b1-9373-c57e8dca38d7.png)